recipes-core/varstored/varstored/0001-add-oxtdb-varstore.patch

```diff
@@ -0,0 +1,864 @@
+################################################################################
+SHORT DESCRIPTION:
+################################################################################
+Add the OpenXT db as a varstored storage mechanism.
+ 
+################################################################################
+LONG DESCRIPTION:
+################################################################################
+varstored can store EFI variables in an arbitrary format. This patch replaces
+the current storage backend (XAPI db) with the OpenXT db. The OpenXT db was
+the preferred choice as it already propagates across host upgrades, which means
+we don't need to worry about carrying EFI guest information over to upgraded
+installs.
+
+Also, remove `chroot` and `unshare` operations in order to avoid granting
+varstored SYS_ADMIN capabilities. We don't need these features and we want to
+keep varstored as limited as possible (SYS_ADMIN would be too permissive).
```

Comment:
I think there is a way to drop SYS_ADMIN, so you could.  Before this patch header, I had wondered if namespaces caused a problem with connecting to DBus IPC.

Reply:
If I dropped SYS_ADMIN elsewhere, wouldn't unshare() and chroot() still fail? Meaning I'd have to get rid of them anyway.

Reply:
I was thinking you do unshare() and chroot() first and then drop SYS_ADMIN afterwards.

Reply:
Per our conversation on Wire, opening a ticket for future evaluation. chroot() had a number of problems, most notably that we couldn't access the db from inside the chroot. unshare() had to be patched to remove a few of the namespaces anyway (as they didn't apply), and the others required varstored to have SYS_ADMIN capabilities, so it was removed altogether.

See ticket: https://openxt.atlassian.net/browse/OXT-1828